src/Feature/Vim/Feature_Vim.re

```diff
@@ -61,17 +61,17 @@ let update = (msg, model: model) => {
 };
 
 module CommandLine = {
-  let getCompletionMeet = commandLine => {
-    let len = String.length(commandLine);
-
-    if (len == 0) {
+  let getCompletionMeet = commandLine =>
+    if (StringEx.isEmpty(commandLine)) {
       None;
     } else {
-      String.index_opt(commandLine, ' ')
-      |> Option.map(idx => idx + 1)  // Advance past space
-      |> OptionEx.or_(Some(0));
+      let meet = StringEx.findUnescapedFromEnd(commandLine, ' ');
+      if (meet == None) {
+        Some(0);
+      } else {
+        meet;
+      };
```

Comment:
This conditional could be streamlined a bit:
```
let maybeMeet = StringEx.findUnescapedFromEnd(commandLine, ' ')
|> OptionEx.or_(Some(0));
```

Reply:
I was looking for Option.or, should have looked for OptionEx, will fix tonight

Reply:
done, resolving